test/unit/modules/rounds.js

```diff
@@ -402,37 +395,81 @@ describe('rounds', () => {
 			});
 
 			it('should set scope.roundFees correctly', () => {
-				return expect(scope.roundFees).to.equal(100);
+				return expect(scope.roundFees).to.equal(0);
 			});
 
 			it('should set scope.roundRewards correctly', () => {
-				return expect(scope.roundRewards).to.deep.equal([1, 2, 3]);
+				return expect(scope.roundRewards).to.deep.equal([0]);
 			});
 
 			it('should set scope.roundDelegates', () => {
 				return expect(scope.roundDelegates).to.deep.equal([
-					'delegate1',
-					'delegate2',
-					'delegate3',
+					scope.block.generatorPublicKey,
 				]);
 			});
 		});
 
-		describe('when summedRound query fails', () => {
-			before(done => {
-				stub = sinon.stub(db.rounds, 'summedRound').rejects('fail');
-				done();
-			});
+		describe('when last block is not genesis block', () => {
+			const scope = { round: 1, block: { height: 2 } };
 
-			after(() => {
-				return stub.restore();
+			describe('when summedRound query is successful', () => {
+				before(done => {
+					var rows = [
+						{
+							rewards: [1.001, 2, 3],
+							fees: 100.001,
+							delegates: ['delegate1', 'delegate2', 'delegate3'],
+						},
+					];
+					stub = sinon.stub(db.rounds, 'summedRound').resolves(rows);
+					done();
+				});
+
+				after(() => {
+					return stub.restore();
+				});
```

Comment:
Would be nice if convert these to `beforeEach` and `afterEach`, I notice difficulties with `before` and `after` maintaining tests in long run.

Reply:
Please do @4miners, the behaviour shouldn't change.